examples/weaviate/personalized_description_with_weaviate_and_gemini_api.ipynb

```diff
@@ -0,0 +1,963 @@
+{
```

Comment:
<p>This is really cool. As a wrap, could you write a short outro too? Just a quick "you learned X, Y, Z" will do, feel free to link out to Weaviate docs or anywhere else relevant (feel free to put your name &amp; a personal link on this too)</p><p> </p><p>Optional, but I'd recommend making it look as good as you can, too. e.g. here you could show the generated output in markdown, embed the image, etc.</p><p> </p>

<pre class="ql-syntax" spellcheck="false">from IPython.display import Markdown

sample, *_ = response.objects

display(Markdown(f"""
&lt;img src="{product.properties['link']}" height=100 /&gt;

{sample.properties["description"]}
"""))
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
<p>Did the best I could with this!</p>